pkg/store/fuzz_test.go

```diff
@@ -0,0 +1,18 @@
+package store
+
+import (
+	"os"
+	"testing"
+)
+
+func FuzzLoadYAMLByFilePath(f *testing.F) {
+	f.Fuzz(func(t *testing.T, fileContents []byte) {
+		err := os.WriteFile("yaml_file.yml", fileContents, 0o600)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer os.Remove("yaml_file.yml")
+
+		_, _ = LoadYAMLByFilePath("yaml_file.yml")
```

Comment:
Isn't the fuzzing function missing an `// Output:` comment?

Reply:
I am not familiar with `// Output:`. Could you make a code suggestion? Or do you have a link to documentation for that?